src/ChunkMap.h

```diff
@@ -377,17 +378,20 @@ class cChunkMap
 	/** Try to Spawn Monsters inside all Chunks */
 	void SpawnMobs(cMobSpawner & a_MobSpawner);
 
+	size_t TickAndUnload(std::chrono::milliseconds a_Dt, int a_UnloadChance);
```

Comment:
Whatever the reason was for the merge and having areturn value, it seems it is no longer justified. I'll change this.